security/zero_trust_workload_identity_manager/zero-trust-manager-release-notes.adoc

```diff
@@ -11,8 +11,204 @@ The {zero-trust-full} leverages Secure Production Identity Framework for Everyon
+==== Custom namespace deployment
+
+* The Operator and all associated operands can now be deployed within a custom namespace, providing flexibility for organizations with specific namespace governance requirements.
+
+[id="zero-trust-manager-1-0-0-postgres-database-support_{context}"]
+==== PostgreSQL database support
+
+SPIRE Server now supports PostgreSQL as an external database backend, accommodating production deployments that necessitate enterprise-grade data persistence and high availability.
+
+* Supported Types: sqlite3 (default), postgres, mysql.
+
+* Customer action required:
+
+** For production, evaluation of migration from SQLite to PostgreSQL is recommended.
+** Creation and configuration of Kubernetes Secrets for database TLS certificates and credentials are required.
+
+[id="zero-trust-manager-1-0-0-proxy-aware-operator-operands_{context}"]
+==== Proxy-awared Operator and operands
+
+* The operator and all managed operands are now proxy-aware and automatically inherit cluster-wide proxy settings when configured.
+
+[id="zero-trust-manager-1-0-0-enhanced-security-context_{context}"]
+==== Enhanced Security Context Constraints
+
+* SPIRE Agent and SPIFFE CSI Driver now operate under the restricted Security Context Constraints (SCC).
+
+* All operand containers are configured with the `ReadOnlyRootFilesystem` set to `true`.
+
+[id="zero-trust-manager-1-0-0-enhanced-api-validation_{context}"]
+==== Enhanced API validation
+
+Comprehensive Common Expression Language (CEL) validation has been integrated into all Custom Resource Definitions (CRDs) to prevent configuration errors during admission control.
+
+* Key validations:
+
+** All Operator CRDs are enforced as singletons (must be named `cluster`).
+** Immutable Fields: Fields including `trustDomain`, `clusterName`, `bundleConfigMap`, federation `bundleEndpoint`, and all `Persistence` settings (`size`, `accessMode`, and `storageClass`) are now immutable after initial creation.
+
+* Customer action required:
+
+** Review existing CR configurations to ensure compliance with the new validation rules.
+
+[id="zero-trust-manager-1-0-0-common-configuration_{context}"]
+==== Common configuration consolidation
+
+* Standard configuration options (`labels`, `resources`, `affinity`, `tolerations`, `nodeSelector`) are now standardized across all operand CRs via a shared `CommonConfig` structure.
+
+[id="zero-trust-manager-1-0-0-immutable-fields_{context}"]
+==== Immutable fields
+
+The following fields are now strictly enforced as *immutable* after initial creation:
+
+[cols="1,1,1",options="header"]
+|===
+| Custom resource
+| Field
+| Notes
+
+| ZeroTrustWorkloadIdentityManager
+| `trustDomain`, `clusterName`, `bundleConfigMap`
+| Cannot be modified post-creation.
+
+| SpireServer
+| `persistence.size`, `persistence.accessMode`, `persistence.storageClass`
+| Cannot be modified post-creation.
+
+| SpireServer
+| `federation.bundleEndpoint`
+| Cannot be removed once configured.
+|===
+
```

Comment:
consider removing this table as they has been mentioned at this line already: https://github.com/openshift/openshift-docs/pull/103679/files#diff-cae3550cca1c8918c4da09faef482503a73e4be67ba89e4e2b5259044e9de140R138